web_src/js/features/repo-projects.js

```diff
@@ -38,18 +42,18 @@ function moveIssue({item, from, to, oldIndex}) {
     })),
   };
 
-  $.ajax({
-    url: `${to.getAttribute('data-url')}/move`,
-    data: JSON.stringify(columnSorting),
-    headers: {
-      'X-Csrf-Token': csrfToken,
-    },
-    contentType: 'application/json',
-    type: 'POST',
-    error: () => {
-      from.insertBefore(item, from.children[oldIndex]);
-    },
-  });
+  try {
+    const response = await POST(`${to.getAttribute('data-url')}/move`, {
+      data: columnSorting,
+    });
+
+    if (!response.ok) {
+      throw new Error('Failed to move issue');
```

Comment:
I'd like some more info in this message such as
```suggestion
      throw new Error(`Failed to move issue - received status: ${response.status} ${response.statusText}`);
```

Reply:
@silverwind asked for these to be removed so I committed https://github.com/go-gitea/gitea/pull/29814/commits/0520d2f6c5c1b2c87fe0b4abfec9f1da1acf76cd